launchdarkly/README.md

```diff
@@ -1,45 +1,56 @@
 # LaunchDarkly Integration
 
-## Overview
+### Overview
 
-This integration tracks metrics from [LaunchDarkly's relay proxy][1], such as number of proxied stream connections and proxied API route requests.
+Deploying features with LaunchDarkly is great, but it can be difficult to see
+the effects of these deployments on users' services or systems. Datadog's
+monitoring and analytics tools for modern DevOps teams allow their customers to
+observe the health of their systems over time. The LaunchDarkly integration with
+Datadog exposes LaunchDarkly events in context with events gathered by the
+Datadog agents. This makes it easier to reason about changes to customers'
+systems as a result of feature deployments. For example, if a feature gets
+deployed that causes a service to slow down, a DevOps engineer can see the cause
+within Datadog.
 
-## Setup
+![LaunchDarkly events in Datadog][2]
 
-### Configuration
+### Setup
 
-After you've configured the [relay proxy][2], add the following section to your LaunchDarkly relay instance's `ld-relay.conf` file:
+#### Configuration
 
-```
-[datadog]
-enabled=true
-statsAddr="YOUR_STATS_ADDRESS"
-```
+This integration will need a [Datadog API key][1]. Only Datadog admins can
+create an API key. Once you've obtained a Datadog API key, visit the
+[LaunchDarkly documentation for the Datadog integration][3] for further
+instructions on how to register the API key in LaunchDarkly.
```

Comment:
```suggestion
Visit the [LaunchDarkly documentation for the Datadog integration][3] to learn how to setup the Datadog-LaunchDarky integration.

**Note**: In order to set it up, you need a valid [Datadog API key][1].
```

All instructions are already outlined in the LaunchDarkly doc, no need to duplicate them here :)